extras/ams_wrapper/src/api/types.py

```diff
@@ -80,29 +77,30 @@ def as_dict(self):
 
 
 class SingleClassification:
-    def __init__(self, attributes: List[Attribute]):
+    def __init__(self, subtype_name: str, attributes: List[Attribute]):
         self.attributes = attributes
+        self.type_name = "classification"
+        self.subtype_name = subtype_name
 
     def as_dict(self):
-        result_dict = {}
-        if self.attributes is not None:
-            result_dict["attributes"] = [attribute.as_dict()
-                                         for attribute in self.attributes]
+        result_dict = {
+            "type": self.type_name,
+            "subtype": self.subtype_name,
+            "classification": self.attributes[0].as_dict()
+        }
         return result_dict
 
 
 class Classification(ResultType):
-    def __init__(self, subtype_name: str, classifications: List[SingleClassification]):
-        self.type_name = "classification"
-        self.subtype_name = subtype_name
+    def __init__(self, classifications: List[SingleClassification]):
         self.classifications = classifications
 
     def as_dict(self):
         result_dict = {
-            "type": self.type_name,
-            "subtype": self.subtype_name,
-            "classifications": [classification.as_dict() for classification in self.classifications]
+            "inferences": [classification.as_dict() for classification in self.classifications]
         }
+        result_dict["inferences"][0], result_dict["inferences"][1] = \
+            result_dict["inferences"][1], result_dict["inferences"][0]
```

Comment:
What's the point of that switch?

Reply:
> What's the point of that switch?

This switching is just to match with the order of desired schema output(Like the order of "type" ,"color"). But going forward , it can be removed since the order might change for mode to model.